app/services/ReviewService.scala

```diff
@@ -147,4 +115,56 @@ class ReviewService @Inject() (publishedService: PublishedService, repository: A
         }
     }
 
+  private def changeStatus(id: String, status: String, userId: String, reviewType: String): Future[RequestOutcome[Unit]] =
+    repository.changeStatus(id, status, userId) map {
+      case Left(Errors(DatabaseError :: Nil)) =>
+        logger.error(s"$reviewType - database error changing status")
+        Left(Errors(InternalServiceError))
+      case error@Left(Errors(NotFoundError :: Nil)) =>
+        logger.warn(s"$reviewType: Change Status: process $id was not found")
+        error
+      case Left(errors) =>
+        logger.error(s"$reviewType: changeStatus: for $id - $errors")
+        Left(errors)
+      case Right(_) => Right(())
+    }
+
+  private def getContentToUpdate(id: String, expectedStatus: String): Future[RequestOutcome[ApprovalProcess]] = repository.getById(id) map {
+    case Right(process) if process.meta.status == expectedStatus => Right (process)
+    case Right(process) =>
+      logger.warn(s"Invalid Process Status Change requested for process $id: " +
+        s"Expected Status: '$expectedStatus' Status Found: '${process.meta.status}'")
+      Left(Errors(StaleDataError))
+    case Left(errors) =>
+      logger.warn(s"getContentToUpdate - error retrieving process $id - error returned $errors.")
+      Left(errors)
+  }
+
+  private def getReviewInfo(id: String, reviewType: String, version: Int): Future[RequestOutcome[ProcessReview]] = {
```

Comment:
approvalReviewInfo and getReviewInfo both return an instance of ProcessReview. getReviewInfo has the added argument version. Do we need both methods?

Reply:
approvalReviewInfo is called from the controller and only has those 2 args, getReviewInfo is a helper method that is called in more than one place (and is passed the version after the ApprovalProcess is retrieved)